src/patternfly/components/Tabs/tabs.hbs

```diff
@@ -7,7 +7,7 @@
   {{~#if tabs--HasNoBorderBottom}} pf-m-no-border-bottom{{/if}}
   {{~#if __tabs-list--IsOverflow}} pf-m-overflow{{/if}}
   {{~#if tabs--IsAnimateCurrent}} pf-m-animate-current{{/if}}
-  {{~#if tabs--isNav}} pf-m-nav{{/if}}
+  {{~#if tabs--IsNav}} pf-m-nav{{/if}}
```

Comment:
This depends what the intended structure is, but we should ensure a `nav` element wraps the Tab being used. It would either live in this file or in the compass_nav file.

If it lived here it could be a followup since we don't currently support it, but we would want to update the wrapper element to be a nav element when IsNav is true (rather than needing to also pass ing `tabs--type="nav"`). Would also need to not apply the role of region on line 16 when IsNav is true so as to not override the nav semantics.

Might make sense to live here so that it could go into React as well.